src/NetTopologySuite/Geometries/Implementation/PackedCoordinateSequence.cs

```diff
@@ -395,7 +395,7 @@ public PackedFloatCoordinateSequence(double[] coords, int dimension, int measure
         {
             if (coords == null)
             {
-                _coords = new float[0];
+                _coords = Array.Empty<float>();
```

Comment:
:page_facing_up: Same basic note about behavior change as in `MinimumBoundingCircle.cs`